test/core/modules1/canvas-resize.js

```diff
@@ -0,0 +1,20 @@
+var test = require('tape');
```

Comment:
This is odd, why is this tested differently? Can we add a comment as to why, as it may confuse future developers. Thanks!

Reply:
Ya sure!! These tests actually follow from @HarshKhandeparkar 's work.
Maybe you can tell Harsh why we have it different in the modules test...